lib/cdo/delete_accounts_helper.rb

```diff
@@ -82,6 +82,10 @@ def clean_level_source_backed_progress(user_id)
     updated_rows = GalleryActivity.where(user_id: user_id).update_all(level_source_id: nil)
     @log.puts "Cleaned #{updated_rows} GalleryActivity" if updated_rows > 0
 
+    @log.puts "Cleaning AssessmentActivity"
+    updated_rows = AssessmentActivity.where(user_id: user_id).update_all(level_source_id: nil)
+    @log.puts "Cleaned #{updated_rows} AssessmentActivity" if updated_rows > 0
```

Comment:
Sorry I'm late on the review for this.  I'd appreciate a test case for this in [DeleteAccountsHelperTest](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/helpers/delete_accounts_helper_test.rb#L446-L465).